pkg/cmd/update-tls-artifacts/generate-owners/generate_owners_flags.go

```diff
@@ -68,11 +67,9 @@ func (f *GenerateOwnersFlags) Validate() error {
 
 func (f *GenerateOwnersFlags) ToOptions() (*GenerateOwnersOptions, error) {
 	return &GenerateOwnersOptions{
-		TLSInfoDir: f.TLSInfoDir,
-		Verify:     f.Verify,
-		Requirements: []tlsmetadatainterfaces.Requirement{
-			ownership.NewOwnerRequirement(),
-		},
+		TLSInfoDir:   f.TLSInfoDir,
+		Verify:       f.Verify,
+		Requirements: tlsmetadatadefaults.GetDefaultTLSRequirements(),
```

Comment:
`Default` here is ambiguous - it should be all requirements imo. Its useful to have a list of all violations, but only some are enforced in tests